lib/Stan_math_code_gen.ml

```diff
@@ -20,15 +36,17 @@ let%expect_test "emit function type raises" =
   | exception e -> print_s [%sexp (e : exn)] ) ;
   [%expect {| ((x UMathLibraryFunction) "not implemented yet") |}]
 
-let pp_call ppf (name, pp_args, args) =
-  pf ppf "%s(@[<hov>%a@])" name pp_args args
+let pp_call ppf (name, pp_arg, args) =
+  pf ppf "%s(@[<hov>%a@])" name (list ~sep:comma pp_arg) args
```

Comment:
Is this enough to deal with functions which take some arguments which are strings are others that are expressions, which is allowed in Stan, like
`
print("hello", {1, 2, 3});
`.
Arguably, we should just add a string type to the language and treat strings are expressions.

Reply:
If I understand you right, that's already happening in the MIR and Ast_to_Mir - printables become Lit(Str, unquote p) and expressions are also translated correctly, and we print string literals correctly.

Reply:
Right! Thanks! That makes sense.